presets/4.3/other/Tehllama_5in_2350-2750KV-Freedom_3S-4S-5S_Race_Tune.txt

```diff
@@ -0,0 +1,245 @@
+set simplified_dterm_filter_multiplier = 140
+
+profile 1
+# profile 1 - 5S
+set dterm_lpf1_dyn_min_hz = 98
+set dterm_lpf1_dyn_max_hz = 277
+set dterm_lpf1_dyn_expo = 10
+set dterm_lpf1_static_hz = 0
+set dterm_lpf2_static_hz = 210
+set vbat_sag_compensation = 77
+set iterm_rotation = ON
+set iterm_relax_cutoff = 33
+set yaw_lowpass_hz = 115
+set throttle_boost = 2
+set p_pitch = 42
+set i_pitch = 63
+set d_pitch = 34
+set f_pitch = 134
+set p_roll = 36
+set i_roll = 55
+set d_roll = 27
+set f_roll = 116
+set p_yaw = 36
+set i_yaw = 55
+set f_yaw = 116
+set d_min_roll = 21
+set d_min_pitch = 26
+set d_max_advance = 0
+set motor_output_limit = 92
+set auto_profile_cell_count = 5
+set feedforward_jitter_factor = 12
+set feedforward_boost = 8
+set throttle_boost = 5
+set feedforward_max_rate_limit = 103
+set dyn_idle_min_rpm = 24
+set dyn_idle_p_gain = 42
+set dyn_idle_i_gain = 42
+set dyn_idle_d_gain = 42
+set simplified_master_multiplier = 65
+set simplified_i_gain = 85
+set simplified_d_gain = 110
+set simplified_pi_gain = 125
+set simplified_dmax_gain = 85
+set simplified_feedforward_gain = 150
+set simplified_pitch_d_gain = 110
+set simplified_pitch_pi_gain = 110
+set simplified_dterm_filter = OFF
+set simplified_dterm_filter_multiplier = 140
+
+profile 2
+# profile 2 - 3S Spec
+set dterm_lpf1_dyn_min_hz = 98
+set dterm_lpf1_dyn_max_hz = 222
+set dterm_lpf1_dyn_expo = 7
+set dterm_lpf1_static_hz = 0
+set dterm_lpf2_static_hz = 210
+set vbat_sag_compensation = 99
+set iterm_relax = RPY
+set iterm_relax_cutoff = 33
+set yaw_lowpass_hz = 115
+set throttle_boost = 12
+set p_pitch = 64
+set i_pitch = 98
+set f_pitch = 220
+set p_roll = 59
+set i_roll = 89
+set d_roll = 42
+set f_roll = 201
+set p_yaw = 59
+set i_yaw = 89
+set f_yaw = 201
+set d_min_roll = 33
+set d_min_pitch = 35
+set d_max_advance = 0
+set motor_output_limit = 81
+set auto_profile_cell_count = 3
+set feedforward_jitter_factor = 12
+set feedforward_boost = 18
+set throttle_boost = 18
+set feedforward_max_rate_limit = 103
+set dyn_idle_min_rpm = 24
+set dyn_idle_p_gain = 42
+set dyn_idle_i_gain = 42
+set dyn_idle_d_gain = 42
+set simplified_master_multiplier = 105
+set simplified_i_gain = 85
+set simplified_d_gain = 105
+set simplified_pi_gain = 125
+set simplified_dmax_gain = 85
+set simplified_feedforward_gain = 160
+set simplified_pitch_d_gain = 95
+set simplified_pitch_pi_gain = 105
+set simplified_dterm_filter = OFF
+
+
+#$ OPTION BEGIN (UNCHECKED): Apply TPA Settings to all rateprofiles
+rateprofile 0
+set tpa_rate = 72
+set tpa_breakpoint = 1250
+
+rateprofile 1
+set tpa_rate = 72
+set tpa_breakpoint = 1250
+
+rateprofile 2
+set tpa_rate = 72
+set tpa_breakpoint = 1250
+
+rateprofile 3
+set tpa_rate = 72
+set tpa_breakpoint = 1250
+
+rateprofile 4
+set tpa_rate = 72
+set tpa_breakpoint = 1250
+
+rateprofile 5
+set tpa_rate = 72
+set tpa_breakpoint = 1250
+
+rateprofile 0
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Apply TPA Settings to current rateprofile only
```

Comment:
why you want TPA for the current profile only?

Reply:
Realistically a somewhat vestigial option - but I wanted to provide a choice where if somebody had 'race' rates on a setup, and different cinematic rates, it would leave every other profile alone.  I can't actually imagine a use case where that is relevant, but wanted to leave an option.

Reply:
I finally remembered why - whenever it applies rateprofile changes to ALL rate profiles, it drops the user back on RateProfile 0, because of how it works.  If that is an undesired behavior, users can select 'current profile only'